File: ui/src/main/java/com/midtrans/sdk/corekit/models/CustomerDetails.java
package com.midtrans.sdk.corekit.models;

/**
 * It holds an information about customer like name , email and phone.
 *
 * Created by shivam on 10/29/15.
 */
public class CustomerDetails extends com.midtrans.sdk.uikit.api.model.CustomerDetails {
    public CustomerDetails(String customerIdentifier, String firstName, String lastName, String email, String phone, ShippingAddress shippingAddress, BillingAddress billingAddress) {
        super(customerIdentifier, firstName, lastName, email, phone, shippingAddress, billingAddress);
    }
}

File: ui/src/main/kotlin/com/midtrans/sdk/uikit/api/model/CustomerDetails.kt
package com.midtrans.sdk.uikit.api.model

import com.midtrans.sdk.corekit.api.model.CustomerDetails


class CustomerDetails(
    customerIdentifier: String? = null,
    firstName: String? = null,
    lastName: String? = null,
    email: String? = null,
    phone: String? = null,
    shippingAddress: Address? = null,
    billingAddress: Address? = null
) : CustomerDetails(
    customerIdentifier = customerIdentifier,
    firstName = firstName,
    lastName = lastName,
    email = email,
    phone = phone,
    shippingAddress = shippingAddress,
    billingAddress = billingAddress
)
